service/service.go

```diff
@@ -217,13 +251,17 @@ func (svc *Service) Close(ctx context.Context) error {
 }
 
 // registerCheckers adds the checkers for the service clients to the health check object.
-func (svc *Service) registerCheckers() error {
+func (svc *Service) registerCheckers() (err error) {
```

Comment:
Was re-adding the named return variable on purpose?

Reply:
no, it was a mistake, thanks for spotting it!